framework/src/utils/PerfGraphLivePrint.C

```diff
@@ -135,10 +143,14 @@ PerfGraphLivePrint::printStats(PerfGraph::SectionIncrement & section_increment_s
       std::chrono::duration<double>(section_increment_finish._time - section_increment_start._time)
           .count();
 
-  auto memory_increment = section_increment_finish._memory - section_increment_start._memory;
+  auto memory_total = section_increment_finish._memory;
 
   auto num_horizontal_chars = message.size() + (2 * section_increment_start._print_stack_level);
 
+  // Add multiapps prefix size
+  if (!_app.isUltimateMaster())
+    num_horizontal_chars += _app.name().size() + 2;
```

Comment:
Is this hard-coded two going to work for nested mulitiapps?

Reply:
yes it does! even added a test for it